docs/proposals/node-removal.md

```diff
@@ -0,0 +1,72 @@
+# Proposal: Node remove
+
+Last updated: 19.03.2021
+
+## Abstract
+
+Need to determine actions for CSI for node removing process
+
+## Proposal
+
+If user aims to delete node from cluster, he/she must perform following steps for CSI.  
+
+###Node is healthy
+
+####After node drain
+1) Determine removing Node UUID 
+2) Delete PVCs used by Pods, which were run on deleted node
+
+###Node is unhealthy
+
+####Before node deletion: 
+1) Determine removing Node UUID 
+
+####After node deletion: 
+1) Delete PVCs used by Pods, which were run on deleted node
+
+###Common actions
+####After node deletion: 
+1) Patch according Volumes custom resources with empty finalizer  
+```
+kubectl get volume -o jsonpath='{range .items[?(.spec.NodeId == "<node_uuid>")]}{@.metadata.name}{" "}' | xargs kubectl patch volume --type merge -p '{"metadata":{"finalizers":null}}'
```

Comment:
Is it possible to expose node UUID and other important values by default to avoid complex commands?